install.sh

```diff
@@ -0,0 +1,443 @@
+#!/bin/bash
+THIS_VERSION="1.0"
```

Comment:
```suggestion
THIS_VERSION="1.0.0"
```
We may want to go with a full semantic versioning here.